tests/basic/podmetric_test.go

```diff
@@ -24,11 +27,14 @@ var _ = Describe("{PodMetricFunctional}", func() {
 	var testrailID, runID int
 	var contexts []*scheduler.Context
 	var namespacePrefix string
+	var meteringIntervalString = os.Getenv("PODMETRIC_METERING_INTERVAL_MINUTES")
+	var callHomeIntervalString = os.Getenv("PODMETRIC_METERING_INTERVAL_MINUTES")
```

Comment:
is it intentional for these to be the same env var?

Reply:
yes. i guess in that case, i can load once and assign the var to the other?

Reply:
No this is good, maybe just include a comment on why we're setting the metering/callhome interval to be the same. Others looking at this later on might see it as a bug.